python/ola/PidStore.py

```diff
@@ -157,8 +157,34 @@ def ValidateAddressing(self, args, command_class):
         return False
     return True
 
-  def __cmp__(self, other):
-    return cmp(self._value, other._value)
+  def __eq__(self, other):
+    if other.__class__ is not self.__class__:
+      return False
+    return self.value == other.value and \
```

Comment:
From https://github.com/python/cpython/blob/0baa72f4b2e7185298d09cf64c7b591efcd22af0/Lib/asyncio/events.py#L97-L145 it looks like round brackets means we can drop the backslash and fix the self. alignment.